ipv-core/src/test/resources/localization/test.swift.codes.csv

```diff
@@ -0,0 +1,9 @@
+AAAACAAA
```

Comment:
Maybe we can try testing some SWIFT code that is 11 characters long?

Reply:
Good idea - changing some of the loaded test values to 11 character format